configure.sh

```diff
@@ -14,8 +13,9 @@ function usage()
 	echo "The following options can be set, with defaults specified in brackets:"
 	echo "  --target MODE         Sets the build mode. Supported build modes are"
 	echo "                        'debug' and 'release'. [${TARGET}]"
-	echo "  --circt-path PATH     Sets the path for the CIRCT tools and enables"
-	echo "                        building with CIRCT support. [${CIRCT_PATH}]"
+	echo "  --enable-hls          Enable the HLS backend, which depends on CIRCT."
+	echo "  --circt-path PATH     Sets the path for the CIRCT tools."
+	echo "                        [${CIRCT_PATH}]"
```

Comment:
Could we merge these two flags into one? In other words, `--enable-hls PATH`. One of the flags does not make sense without the other.

Reply:
This would also render the need for the circt path mandatory, which means the CIRCT_PATH can be initialized with `""`.

Reply:
This is intentional. The whole point is that only advanced users should need to think about where the build happens. Similar to not having to think about that there is a build-release and build-debug directory.
So, instead of merging them, I would argue for removing --circt-path completely and always force it to be in one specific location.